recipe/bld.bat

```diff
@@ -1,10 +1,10 @@
 setlocal EnableDelayedExpansion
 
-:: brand Python with conda-forge startup message
-%SYS_PYTHON% %RECIPE_DIR%\brand_python.py
-if errorlevel 1 exit 1
+REM brand Python with conda-forge startup message
+REM %SYS_PYTHON% %RECIPE_DIR%\brand_python.py
+REM if errorlevel 1 exit 1
```

Comment:
This is needed for conda-forge, unless a patch added similar messaging.

Reply:
Yeah, how can we try to keep our recipes in sync. What do you think about using env vars like I do for static vs dynamic Python in meta.yaml? I'm OK with doing something like that personally.

Reply:
Sounds cleaner